src/material/datepicker/date-range-input.ts

```diff
@@ -123,11 +170,8 @@ export class MatDateRangeInput<D> implements MatFormFieldControl<DateRange<D>>,
   /** Value for the `aria-labelledby` attribute of the inputs. */
   _ariaLabelledBy: string | null = null;
 
-  /** Placeholder for the start input. */
-  @Input() startPlaceholder: string;
-
-  /** Placeholder for the end input. */
-  @Input() endPlaceholder: string;
+  /** Date selection model currently registered with the input. */
+  private _model: MatDateSelectionModel<DateRange<D>, D> | undefined;
```

Comment:
nit: I think you can leave off the `, D` and it will be inferred

Reply:
Ah nice. Cleaned up in a couple of other places.